src/D2L.Security.OAuth2/Keys/Default/MonoRsaSecurityKey.cs

```diff
@@ -0,0 +1,123 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IdentityModel.Tokens;
+using System.Linq;
+using System.Security.Cryptography;
+using System.Text;
+
+namespace D2L.Security.OAuth2.Keys.Default {
+
+	internal sealed class MonoRsaSecurityKey : AsymmetricSecurityKey {

```

Comment:
This class is used else where and would require if directives. I could do that, but it makes it a little bit of a pain to edit. Not sure if we care about a bit of dead code on windows or not.

I can certainly add it, if you feel it's beneficial.